### PR TITLE
htop: fix build dependencies and add --HEAD option

### PR DESCRIPTION
**Pull request by s3ishr**
### All Submissions:

- [x] Have you followed the guidelines in our [Contributing](https://github.com/Homebrew/homebrew/blob/master/.github/CONTRIBUTING.md) document?
- [x] Have you checked to ensure there aren't other open [Pull Requests](https://github.com/Homebrew/homebrew/pulls) for the same update/change?

htop don't need autoconf, automake and libtool on release tarball.

### CODE REVIEW

File: Library/Formula/htop.rb [hunk 1]
```diff
@@ -1,26 +1,25 @@
 class Htop < Formula
   desc "Improved top (interactive process viewer)"
-  homepage "https://github.com/hishamhm/htop"
-  url "https://github.com/hishamhm/htop/archive/2.0.1.tar.gz"
-  sha256 "636c1e8b703058e793e8d25423af4b74059290ef9e48fa261ba58555069517b5"
-
-  bottle do
```
Comment (rakkesh) [inline Library/Formula/htop.rb]: There is no need to remove the `bottle do`  block.

File: Library/Formula/htop.rb [hunk 2]
```diff
@@ -1,26 +1,25 @@
...
-    sha256 "b8afe978bc97ef2d18767c769941b81e0501c20566d2427e2e421bb69ca94e5a" => :el_capitan
-    sha256 "f667485b77c96a1c7fe4923858cb60f1d5700412a1921dd9a12ac4fb50f305f2" => :yosemite
-    sha256 "47fd6612ee8889637c3aa47a6c8cdc5812b8b8f8fa4853f35f2131bb8da43884" => :mavericks
+  homepage "http://hisham.hm/htop/"
+  url "http://hisham.hm/htop/releases/2.0.1/htop-2.0.1.tar.gz"
```
Comment (DomT4) [inline Library/Formula/htop.rb]: Please use `https://` for both these links.

File: Library/Formula/htop.rb (hunk 1 above)
Comment (DomT4) [inline Library/Formula/htop.rb]: :+1:

**Comment by apjanke** [comment]
Closed by https://github.com/Homebrew/homebrew-core/pull/13, which contains these changes and the requested adjustments. Do `brew update; brew upgrade` to pick it up.

Thank you for your contribution to Homebrew!

**Pull request closed without merging**